docs/GettingStarted.md

```diff
@@ -50,89 +50,57 @@ To enable the Rails auto instrumentation, create an initializer file in your ``c
-* ``auto_instrument_grape``: if set to ``true`` and you're using a Grape application, all calls to your endpoints are
-  traced, including filters execution.
-* ``default_service``: set the service name used when tracing application requests. Defaults to ``rails-app``
-* ``default_controller_service``: set the service name used when tracing a Rails action controller. Defaults to ``rails-controller``
-* ``default_cache_service``: set the cache service name used when tracing cache activity. Defaults to ``rails-cache``
-* ``default_database_service``: set the database service name used when tracing database activity. Defaults to the
-  current adapter name, so if you're using PostgreSQL it will be ``postgres``.
-* ``distributed_tracing_enabled``: enable [distributed tracing](#Distributed_Tracing) so that this service trace is
+* ``service_name``: set the service name used when tracing application requests (on the `rack` level). Defaults to ``<app_name>``, where `<app_name>` is automatically inferred from your Rails application namespace;
+* ``controller_service``: set the service name used when tracing a Rails action controller. Defaults to ``<app_name>-controller``
+* ``cache_service``: set the cache service name used when tracing cache activity. Defaults to ``<app_name>-cache``
+* ``database_service``: set the database service name used when tracing database activity. Defaults to the
+  current adapter name, so if you're using PostgreSQL it will be ``<app_name>-postgres``.
+* ``distributed_tracing``: enable [distributed tracing](#Distributed_Tracing) so that this service trace is
   connected with a trace of another service if tracing headers are sent
-* ``default_grape_service``: set the service name used when tracing a Grape application mounted in your Rails router.
-  Defaults to ``grape``
 * ``template_base_path``: used when the template name is parsed in the auto instrumented code. If you don't store
   your templates in the ``views/`` folder, you may need to change this value
 * ``tracer``: is the global tracer used by the tracing application. Usually you don't need to change that value
   unless you're already using a different initialized ``tracer`` somewhere else
-* ``debug``: set to true to enable debug logging.
-* ``trace_agent_hostname``: set the hostname of the trace agent.
-* ``trace_agent_port``: set the port the trace agent is listening on.
-* ``env``: set the environment. Rails users may set it to ``Rails.env`` to use their application settings.
-* ``tags``: set global tags that should be applied to all spans. Defaults to an empty hash
 
-#### Disabling Rails auto-instrumentation
+To integrate Rails instrumentation with third-party libraries such as Grape, please check the available settings below.
 
-If you wish to disable all Rails auto-instrumentation, you need to set the env var ``DISABLE_DATADOG_RAILS``.
+#### Configure the tracer with initializers
```

Comment:
I would prefer making this section independent from Rails. Maybe should we add it at the beginning of the README, or in the advanced section (though it's not really advanced and it's very likely you have to change something like the hostname).

Reply:
@palazzem Moved that to `Advanced Usage`. I totally agree we should find a better place for it, but let's leave that for the next iteration in which we'll effectively restructure the entire documentation. For now, we're at least giving a hint on how the tracer should be configured in the `Quickstart` section.

Reply:
Agreed. Let's move it in Advanced Usage.